src/components/MoneyRequestConfirmationList.js

```diff
@@ -777,12 +825,15 @@ export default compose(
             key: ({policyID}) => `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
             selector: DistanceRequestUtils.getDefaultMileageRate,
         },
-        draftTransaction: {
+        splitTransactionDraft: {
```

Comment:
NAB, this seems unused and can be removed

Reply:
O will see to it.